overlord/managers_test.go

```diff
@@ -718,9 +718,15 @@ apps:
 }
 
 func (s *mgrsSuite) TestHappyRemoveWithQuotas(c *C) {
+	// TODO: we need a variant of this test which disables the memory cgroup
+	// after installing the snap to ensure that the snap can be removed
+	// successfully when the memory cgroup is disabled
 	r := systemd.MockSystemdVersion(248, nil)
```

Comment:
This may just be a clarification and something to learn for me but in previous cgroups usage the required systemd version is 230 as opposed to 248 here and in quota handlers test. Any specific reason?

Reply:
No, just when this test was originally written last year that was the version I had on my machine so I used it as a value that was newer than the version we need